src/hangman/models/game.model.ts

```diff
@@ -100,13 +111,13 @@ export class Game extends AggregateRoot {
     this.playerId = event.playerId;
     this.wordToGuess = event.wordToGuess;
     this.maxGuesses = event.maxGuesses;
-    this.lettersGuessed = [];
+    // this.lettersGuessed = []; // Is this necessary?
     this.dateCreated = event.dateCreated;
     this.dateModified = event.dateModified;
   }
 
   onLetterGuessedEvent(event: LetterGuessedEvent) {
     this.lettersGuessed.push(event.letter[0]);
-    this.dateModified = event.dateModified;
+    this.dateModified = event.dateModified; // FIXME: date modified not passed from command handler
```

Comment:
if you set new Date here, on replaying of all events the dateModified is when the events where replayed in stead of when the action was done by the user

Reply:
I think I understand... so the date modified field is never explicitly passed as a parameter (i.e. when the event is emitted here: `const event = new LetterGuessedEvent(this.id, letter);` )

So when you replay, you want the date modified field to reflect the current date, not the original?

Reply:
the dateModified is the date the user made a change to the Game. So it's stored in the event when the event is generated. 
On replay, date modified field should reflect the date that's stored in the event. 
(as I'm writing this, I realise in the EventInstantiator that deserialises the json event might re-set the date now? If so I'll change that, it should take on the date from the actual json-event coming from eventStore)

Reply:
on replay take over the dateModified from the event